### PR TITLE
V2

### PR DESCRIPTION
**Pull request by vladucu**
**JIRA Ticket:** PROD-1198

# Description

This pulls in upstream `ember-polaris@v6`.

Most of the stuff in here was reviewed by pieces in `ember-polaris`. The biggest chunk of changes that `ember-smile-polaris` has over `ember-polaris` can be seen in [this commit](https://github.com/smile-io/ember-smile-polaris/pull/35/commits/fb9b9b7c37067a7c0c9aa6eb9091ad3121313353) _(though not 💯 of them)_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,93 @@
-# ember-polaris Changelog
+# Changelog
+
+## [Unreleased](https://github.com/smile-io/ember-polaris/tree/HEAD)
+
+[Full Changelog](https://github.com/smile-io/ember-polaris/compare/v5.2.1...HEAD)
+
+:boom: Breaking Change
+
+- Misc updates for v6 [\#620](https://github.com/smile-io/ember-polaris/pull/620) ([vladucu](https://github.com/vladucu))
```
Comment (andrewpye) [inline CHANGELOG.md:9]: This comment is a bit misleading given that it's referring to v6 of `ember-polaris`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,93 @@
-# ember-polaris Changelog
+# Changelog
+
+## [Unreleased](https://github.com/smile-io/ember-polaris/tree/HEAD)
+
+[Full Changelog](https://github.com/smile-io/ember-polaris/compare/v5.2.1...HEAD)
```
Comment (andrewpye) [inline CHANGELOG.md:5]: These (and a bunch of others in this changelog) are referring to `ember-polaris` - is that what we want here?

File: docs/action-list.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+[< Back to Components List](../README.md#components)
```
Comment (andrewpye) [inline docs/action-list.md:1]: Was adding all of these docs files intentional?

File: tests/stubbers/math/random.js [hunk 4]
```diff
@@ -26,7 +24,6 @@ const MathRandomStubber = EmberObject.extend({
   /**
    * Original Math.random to restore later.
    *
-   * @property originalMathRandom
    * @private
    * @type {function}
    */
```
Comment (andrewpye) [inline tests/stubbers/math/random.js:29]: Can't comment on the line but it looks like we've got some `.get()` calls in the codebase still (line 41 etc.), not sure if we want to update those?

Comment (vladucu) [reply tests/stubbers/math/random.js:29]: good catch!
I was thinking we'll run the codemod at some point on the entire repo, not too worried about this personally

File: docs/action-list.md (hunk 3 above)
Comment (vladucu) [reply docs/action-list.md:1]: they are in polaris, so they've been pulled in.... any reason we don't want them?

File: CHANGELOG.md (hunk 2 above)
Comment (vladucu) [reply CHANGELOG.md:5]: will update the changelong before merging...

File: docs/action-list.md (hunk 3 above)
Comment (andrewpye) [reply docs/action-list.md:1]: I don't think so, was just wondering if there was a reason we didn't have them previously 😛

Comment (vladucu) [reply docs/action-list.md:1]: yeah, don't really remember
might have been just a bad merge from upstream at some point 🤷‍♂️

File: CHANGELOG.md (hunk 2 above)
Comment (vladucu) [reply CHANGELOG.md:5]: actually, I see old changelog links to ember-polaris too, so I'd lean to leave as is
not worth the time

**Pull request merged**